index.html

```diff
@@ -0,0 +1,24 @@
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Bowling Scorecard</title>
+
+  <link rel="stylesheet" href="src/style.css">
+
+  <script src="https://code.jquery.com/jquery-latest.min.js" type="text/javascript"></script>
+  <script src="src/interface.js"></script>
+  <script src="src/Frame.js"></script>
+  <script src="src/Scorecard.js"></script>
+</head>
+
+<body>
+  <h1>Bowling Scorecard</h1>
+  <div id="scores"></div><br><br>
+  <form id="new-roll">
+    <input type="number" min="0" max="10" id="roll">
```

Comment:
Such a good idea to use min and max here! Wish I'd thought of it!

Reply:
hate to jump on this. With the number, is there a way to store their added number then use THAT to set the next min/max value? so you can't select a total over 10 pins?

Reply:
Yeah Ben that is a good idea. I didn't get time to do any input validation but this would have been a nice extra flair if I had!